<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>729</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="1">
       <widget class="QFrame" name="frame">
        <property name="minimumSize">
         <size>
          <width>270</width>
          <height>550</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>220</y>
           <width>271</width>
           <height>321</height>
          </rect>
         </property>
         <property name="title">
          <string>Parameters of Mode</string>
         </property>
         <widget class="QLineEdit" name="NumberOfSteps">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>20</y>
            <width>51</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>20</string>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>91</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>   NumberOfSteps</string>
          </property>
         </widget>
         <widget class="QPushButton" name="Start_AutoScan">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>170</y>
            <width>151</width>
            <height>71</height>
           </rect>
          </property>
          <property name="text">
           <string>StartScan</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>50</y>
            <width>81</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>U, kV</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>80</y>
            <width>81</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>I, mkA</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="U_Auto">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>50</y>
            <width>51</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>70</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="I_Auto">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>80</y>
            <width>51</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>25</string>
          </property>
         </widget>
         <widget class="QLabel" name="Current_U">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>50</y>
            <width>46</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
         <widget class="QLabel" name="Current_I">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>80</y>
            <width>46</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
         <widget class="QLabel" name="pos1">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>20</y>
            <width>47</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="with_rotate">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>140</y>
            <width>81</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>with_rotate</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>140</y>
            <width>61</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>rap off</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="TimeCorrect">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>280</y>
            <width>51</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>30</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>280</y>
            <width>81</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Time correction</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>110</y>
            <width>81</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Exposure, ms</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="Exposure">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>110</y>
            <width>51</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>2000</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_7">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>250</y>
            <width>101</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Intensive compare</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="Compare">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>250</y>
            <width>51</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>50</string>
          </property>
         </widget>
         <widget class="QLabel" name="current_step">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>20</y>
            <width>31</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
         <widget class="QLabel" name="pos2">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>50</y>
            <width>47</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
         <widget class="QLabel" name="pos3">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>80</y>
            <width>47</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
         <widget class="QLabel" name="pos4">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>110</y>
            <width>47</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
         <widget class="QLabel" name="pos5">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>140</y>
            <width>47</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
         <widget class="QLabel" name="pos6">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>170</y>
            <width>47</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_2">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>220</y>
            <width>51</width>
            <height>61</height>
           </rect>
          </property>
          <property name="text">
           <string>1600</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_4">
          <property name="geometry">
           <rect>
            <x>180</x>
            <y>290</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="xray_signal">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>190</y>
            <width>70</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>xray signal</string>
          </property>
         </widget>
        </widget>
        <widget class="QGraphicsView" name="graphicsView">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>40</y>
           <width>270</width>
           <height>141</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777212</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
        </widget>
        <widget class="QPushButton" name="LoadAutoContrast">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>0</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>LoadAC</string>
         </property>
        </widget>
        <widget class="QPushButton" name="SaveAutoContrast">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>SaveAc</string>
         </property>
        </widget>
        <widget class="QProgressBar" name="progressBar">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>190</y>
           <width>251</width>
           <height>21</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
        <widget class="QPushButton" name="handle">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>0</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Manual Control</string>
         </property>
        </widget>
        <zorder>groupBox_2</zorder>
        <zorder>graphicsView</zorder>
        <zorder>LoadAutoContrast</zorder>
        <zorder>progressBar</zorder>
        <zorder>handle</zorder>
        <zorder>SaveAutoContrast</zorder>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QFrame" name="frame_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QFrame" name="frame_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QWidget" name="horizontalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>-1</x>
           <y>-1</y>
           <width>751</width>
           <height>41</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="1">
           <widget class="QPushButton" name="load_image">
            <property name="text">
             <string>loadImage</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="convert_image">
            <property name="text">
             <string>convert</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QComboBox" name="image_format">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <item>
             <property name="text">
              <string>.tif</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>.jpg</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QComboBox" name="comboBox"/>
          </item>
         </layout>
        </widget>
        <widget class="QGroupBox" name="groupBox">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>40</y>
           <width>391</width>
           <height>51</height>
          </rect>
         </property>
         <property name="title">
          <string>Status</string>
         </property>
         <widget class="QLabel" name="ErrorLabel">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>91</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>3200 should be divided into a number of steps</string>
          </property>
         </widget>
         <widget class="QLabel" name="status_of_cam_2">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>20</y>
            <width>81</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>correct work</string>
          </property>
         </widget>
         <widget class="QLabel" name="status_of_motor_2">
          <property name="geometry">
           <rect>
            <x>310</x>
            <y>20</y>
            <width>81</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>correct work</string>
          </property>
         </widget>
         <widget class="QLabel" name="status_of_rap_2">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>20</y>
            <width>81</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>correct work</string>
          </property>
         </widget>
         <widget class="QLabel" name="RAP_2">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>10</y>
            <width>47</width>
            <height>13</height>
           </rect>
          </property>
          <property name="text">
           <string>RAP </string>
          </property>
         </widget>
         <widget class="QLabel" name="Motor_2">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>310</x>
            <y>0</y>
            <width>61</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Motor</string>
          </property>
         </widget>
         <widget class="QLabel" name="CAM_2">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>10</y>
            <width>47</width>
            <height>13</height>
           </rect>
          </property>
          <property name="text">
           <string>CAM</string>
          </property>
         </widget>
        </widget>
        <widget class="QPushButton" name="Calibrate">
         <property name="geometry">
          <rect>
           <x>650</x>
           <y>40</y>
           <width>51</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string>Calibrate</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_3">
         <property name="geometry">
          <rect>
           <x>700</x>
           <y>40</y>
           <width>51</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string>2</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_2">
         <property name="geometry">
          <rect>
           <x>400</x>
           <y>50</y>
           <width>101</width>
           <height>41</height>
          </rect>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="currentText">
          <string notr="true">MltCam</string>
         </property>
         <property name="currentIndex">
          <number>1</number>
         </property>
         <property name="duplicatesEnabled">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>AlphaCam</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>MltCam</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Cam choose</string>
          </property>
         </item>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>776</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
